Documentation/PageTsconfig/Mod/Shared.rst

```diff
@@ -1,182 +1,180 @@
+    'Columns' view of the Web > Page module.
 
-   Used on top of backend layouts, this setting controls which columns are editable. Columns configured
-   in the Backend Layout which are not listed here, will be displayed with placeholder area.
+    Used on top of backend layouts, this setting controls which columns are editable. Columns configured
+    in the Backend Layout which are not listed here, will be displayed with placeholder area.
 
-   Each column has a number which ultimately comes from the configuration of the table tt_content,
-   field 'colPos'. These are the values of the four default columns used in the default backend layout:
+    Each column has a number which ultimately comes from the configuration of the table tt_content,
+    field 'colPos'. These are the values of the four default columns used in the default backend layout:
 
-   Left: `1`, Normal: `0`, Right: `2`, Border: `3`
+    Left: `1`, Normal: `0`, Right: `2`, Border: `3`
 
-:aspect:`Default`
-   1,0,2,3
+..  _example_for_backend_layout:
 
+Example: Create a basic backend layout
+--------------------------------------
 
-:aspect:`Example`
-   .. _example_for_backend_layout:
+The example creates a basic backend layout and sets the "Left" column to be not editable:
 
-   The example creates a basic backend layout and sets the "Left" column to be not editable:
+*   Create a record of type "Backend Layout", for instance in the root page of your website
 
-   *  Create a record of type "Backend Layout", for instance in the root page of your website
+*   Add a title, e.g. "My Layout"
 
-   *  Add a title, e.g. "My Layout"
+*   Use the wizard to create a two column backend layout, the result may look like this:
 
-   *  Use the wizard to create a two column backend layout, the result may look like this:
+    ..  figure:: /Images/ManualScreenshots/List/SimpleBackendLayout.png
+        :alt: A simple backend layout
 
-      .. figure:: /Images/ManualScreenshots/List/SimpleBackendLayout.png
-         :alt: A simple backend layout
+        A simple backend layout
 
-         A simple backend layout
+*   Create a page and select this new backend layout in the "Appearance" tab.
+    The page module then looks like this, displaying the two defined columns:
 
-   *  Create a page and select this new backend layout in the "Appearance" tab.
-      The page module then looks like this, displaying the two defined columns:
+    ..  figure:: /Images/ManualScreenshots/Page/SimpleBackendLayoutInPageModule.png
+        :alt: Backend layout used in page module
 
-      .. figure:: /Images/ManualScreenshots/Page/SimpleBackendLayoutInPageModule.png
-         :alt: Backend layout used in page module
+        Backend layout used in page module
 
-         Backend layout used in page module
+*   Now set the "Left" column to be not editable using page TSconfig in the
+    :guilabel:`Resources` tab of the page, by restricting `colPos_list` to
```

Comment:
When writing "Using ... the "Resources" tab of the page" the caption below in the example should be omitted, as this is then not this file.

Reply:
I did not do any text changes, just changes to indentaion and the confval. This should facislitate backporting